storage/client/types.go

```diff
@@ -2,10 +2,29 @@
 package client
 
 import (
+	"fmt"
 	"math/big"
+	"strings"
 	"time"
 )
 
+type BigInt struct {
+	big.Int
+}
+
+func NewBigInt(v int64) BigInt {
```

Comment:
I'm a medium face on how much this helps. 🤔

Reply:
I find the implementation syntax to be mildly convoluted, so having this does help readability a tiny bit. But I wouldn't be too sorry to see it go, if you prefer.